[Feat] jwt 적용

qordpsem opened the pull request:
## 연관된 이슈
Closes #45
Closes #76
## 작업 내용
- admin db 추가
- 그외 jwt 관련 파일 추가
- user, management, admin 각각 role user, management, admin 가지는 것으로 수정했습니다 기존에 role 이 나뉘어져있는게 복잡해서 자꾸 오류가나서 부득 db 건들였습니다 죄송함다,, (status 는 기존 그대로 유지)
- exception 추가
- 코드 일단 모든 url 다 permitAll 했습니다 혹시 controller 에 로직 추가하게되면 할때마다 SecurityConfig 에 추가해주세요
(/users/** 이런식으로 하면 그 아래것도 다 되는거 아는데 혹시라도 오류 날까봐 그냥 다 넣었습니다!)

+추가 회원 수정,조회 / 소속사 수정, 조회
+소셜로그인 (구글) 프론트에 연결하고 작동되는거 확인 후 백엔드 푸쉬함
## 리뷰 요구사항 (선택)

이거 dev 에 합치는순간 db는 리셋해야할듯 합니다.. 그전에 db 필요한거 있으면 다쓰시고.. 그러고 merge 하겠습니다..
환경변수 업데이트 됐습니다! slack에 올려놨습니다!

Commit:
[Feat] jwt 적용

- admin db 추가
- 그외 jwt 관련 파일 추가
- user, management, admin 각각 role user, management, admin 가지는 것으로 수정했습니다 기존에 role 이 나뉘어져있는게 복잡해서 자꾸 오류가나서 부득 db 건들였습니다 죄송,, (status 는 기존 그대로 유지)
- exception 추가
- 코드 일단 모든 url 다 permitAll 했습니다 혹시 controller 에 로직 추가하게되면 할때마다 SecurityConfig 에 추가해주십쇼,,
(/users/** 이런식으로 하면 그아래것도 다 되는거 아는데 혹시라도 오류 날까봐 그냥 다 넣었습니다!)

ccc700 reviewed (approved):
굿굿입니덩 근데 머지하시면 저는 좀 나중에 pull 받을게용..ㅋㅋ

Commit:
[Feat] user 회원조회, 회원수정 구현

- SecurityConfig에 주소추가
- Update request 작성
- user controller, service 에 수정, 조회 로직 추가

Commit:
[Modify] 상태코드 수정

- 가입 코드 201 로 수정

Commit:
[Feat] Management 회원조회, 수정

- Management controller, service 추가

nk2200 reviewed (approved):
수고링~

Commit:
[Modify] 회원가입, 회원수정 시 postcode  추가

- user entity,service, response 에 postcode 추가

Commit:
[Feat] 소셜로그인 (구글)

- 프론트에 띄우는데 오래걸려서,, 이제 커밋앤푸쉬,,,, 프론트에 정보 띄우기 성공,,

qordpsem committed:
Merge branch 'dev' into feat/45-login_jwt

Commit:
[Feat] user, management 탈퇴

- soft delete 일단 미적용... (추후 할거 다하고 시간남으면 적용...)

Commit:
Merge remote-tracking branch 'origin/feat/45-login_jwt' into feat/45-login_jwt
